apps/web/components/CategoryFilters/Filter.vue

```diff
@@ -118,10 +124,12 @@ function resetPriceFilter() {
 
 const updateFilter = () => {
   for (const filter of filters) {
-    models[filter.id.toString()] = Boolean(filter.selected) ?? false;
+    const filterId = typeof filter.id === 'string' ? filter.id : filter.id.toString();
```

Comment:
<img width="1348" alt="Screenshot 2023-12-12 at 16 22 36" src="https://github.com/plentymarkets/plentyshop-pwa/assets/149154151/6ec4f3f1-b857-4ddb-8fb6-c735b2065edd">
there's no addition to the code from main branch, just linting triggered that realigned the code

Reply:
id can be either number or string, hence the reason for the check
error highlighted on main branch build

Reply:
I have no errors on **main** branch `yarn build`, and still I don't understand why we need that check, it basically says if it is of type string, take it, otherwise transform it into a string, but `'anyString'.toString()` is still a string, so that check seems redundant to me. Let's maybe discuss this tomorrow.